src/LookupTable/components/list/ListTable.tsx

```diff
@@ -82,10 +82,11 @@ const ListTable: React.FC<any> = ({
                     <DownloadButton
                       popUpContent={strings.LOOKUP_TABLE_DOWNLOAD.replace('%1', lookupTable.label)}
                       id={lookupTable.id}
+                      name={lookupTable.label}
                     />
                   </Button.Group>
                 </Table.Cell>
-                <Table.Cell icon="chevron down" collapsing />
+                {/* <Table.Cell icon="chevron down" collapsing /> */}
```

Comment:
No idea what this was for. I'm assuming it was intended as a possible Accordion opener for some future implementation, so just hidden for now.